modules/explanation-of-capabilities.adoc

```diff
@@ -1,11 +1,9 @@
 // Module included in the following assemblies:
 //
-// *post_installation_configuration/cluster-capabilities.adoc
+// *installing/cluster-capabilities.adoc
 
 :_content-type: REFERENCE
 [id="explanation_of_capabilities_{context}"]
 = Optional cluster capabilities in {product-title} {product-version}
 
-Optional cluster capabilities are components that can be disabled during the installation process. Cluster administrators can disable optional cluster capabilities to reduce the resources consumed by a cluster. Currently, cluster Operators provide a cluster capability's features. 
-
-If you want to use the features provided by a disabled cluster capability, you can enable the capability after installation. After a cluster capability is enabled, it cannot be disabled.
+Currently, cluster Operators provide the features for these optional capabilities. The following summarizes the features provided by each capability and what functionality you lose if it is disabled.
```

Comment:
I think this is an older version of this introductory content.

Reply:
Part of this content was moved to the top of the abstract instead.

Reply:
```suggestion
Currently, cluster Operators provide the features for the optional cluster capabilities. The following summarizes the features provided by each capability and what functionality you lose if it is disabled.
```